tests/test_misc.py

```diff
@@ -14,3 +16,54 @@ def test_version():
     assert len(sver) > 0
     assert iver > 0
 
+def test_error_cb():
+    """ Tests error_cb. """
+
+    def error_cb(error_msg):
+        print('OK: error_cb() called')
+        assert error_msg.code() in (confluent_kafka.KafkaError._TRANSPORT, confluent_kafka.KafkaError._ALL_BROKERS_DOWN)
+
+    conf = {'bootstrap.servers': 'localhost:9093', # Purposely cause connection refused error
+            'group.id':'test',
+            'socket.timeout.ms':'100',
+            'session.timeout.ms': 1000, # Avoid close() blocking too long
+            'error_cb': error_cb
+           }
+
+    kc = confluent_kafka.Consumer(**conf)
+
+    kc.subscribe(["test"])
+    kc.poll(timeout=0.001)
```

Comment:
Does not verify that error_cb was actually called.
Maybe construct it like so, without sleep:
```
in test_error_cb:
   seen_error_cb = True
....
while !seen_error_cb:
    kc.poll(timeout=1)
kc.close()
```

Test timeouts will handle the case where no callback was seen